Module-5-Assignment/StudentInsertionTrigger.trigger

```diff
@@ -0,0 +1,76 @@
+/**
+* Trigger to restrict insertion of a student into a class if that class max capacity is reached.
```

Comment:
This trigger is preventing the insertion of student into a class as well as updating the myCount field update the comment accordingly. Also this trigger is for insert as well as update. Update the name accordingly. A good practice while naming triggers is to name them as <SObjectlabel>Trigger. So here the name will be StudentTrigger. Do for other triggers too and update the test classes names accordingly

Reply:
Done